sound/soc/amd/acp/amd-acp63-acpi-match.c

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: GPL-2.0+
```

Comment:
please check with your lawyers, the Intel code was released as GPL-2.0-only.

Reply:
I do agree. Its overlooked. all files will use GPL-2.0 only license . will update it.